packages/@okta/vuepress-theme-prose/components/OnThisPageItem.vue

```diff
@@ -3,17 +3,19 @@
     <a :href="link.path || '#' + link.slug" class="on-this-page-link" :class="{'router-link-active': imActive}" @click.prevent="clickLink">
       <span >{{link.title}}</span>
     </a>
-    <ul v-if="link.children && (iHaveChildrenActive || imActive)">
-      <OnThisPageItem v-for="(childLink, index) in link.children" :key="index" :link="childLink" :activeAnchor=activeAnchor />
+    <ul v-show="link.children && (iHaveChildrenActive || imActive)">
```

Comment:
Why change from `v-if`?

Reply:
not sure why but with v-if child items not displaying active indicator in case child item is selected, it could be related with v-if logic (to remove an element from the dom tree if v-if condition === false). Also, in this case, v-show could be a better choice because it only set display: none and not rerender dom tree, and v-show/v-if in this situation will be using quite often

Reply:
Ok makes sense, thanks